pkgs/development/python-modules/pynvml/default.nix

```
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "pynvml";
+  version = "8.0.4";
+
+  disabled = pythonOlder "3.6";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0pfykj1amqh1rixp90rg85v1nj6qmx89fahqr6ii4zlcckffmm68";
+  };
+
+  doCheck = false;  # no tests in PyPi dist
```

please at least do an import
```suggestion
  doCheck = false;  # no tests in PyPi dist
  pythonImportsCheck = [
    "pynvml"
  ];
```

---

Added.